cmd/minikube/cmd/root.go

```diff
@@ -95,11 +92,12 @@ func Execute() {
 			exit.Message(reason.WrongBinaryWSL, "You are trying to run windows .exe binary inside WSL, for better integration please use Linux binary instead (Download at https://minikube.sigs.k8s.io/docs/start/.). Otherwise if you still want to do this, you can do it using --force")
 		}
 	}
-
-	if runtime.GOOS == "darwin" && detect.IsAmd64M1Emulation() {
-		exit.Message(reason.WrongBinaryM1, "You are trying to run amd64 binary on M1 system. Please use darwin/arm64 binary instead (Download at {{.url}}.)",
-			out.V{"url": notify.DownloadURL(version.GetVersion(), "darwin", "amd64")})
-	}
+	/*
+		if runtime.GOOS == "darwin" && detect.IsAmd64M1Emulation() {
```

Comment:
here lets still Warn them... and we advise to use the native binary...

Reply:
Done